src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -83,6 +83,8 @@ def load_arguments(self, _):
         with self.argument_context(scope) as c:
             c.argument('deployment', options_list=[
                 '--deployment', '-d'], help='Name of an existing deployment of the app. Default to the production deployment if not specified.', validator=validate_deployment_name)
+            c.argument('main_entry', options_list=[
+                '--main-entry', '-m'], help="A string containing the path to the .NET executable relative to zip root.")
```

Comment:
`The path to the .NET executable relative to zip root.`  would be more succinct.